docs/developers/patch.md

```diff
@@ -50,8 +50,8 @@ It's also possible to add left of a token instead of a tree node.
 
 ```scala mdoc
 doc.tokens.collect {
-  case brace @ Token.KwExtends() =>
-    Patch.addLeft(brace, "/* => */ ")
+  case `extends` @ Token.KwExtends() =>
+    Patch.addLeft(`extends`, "/* => */ ")
```

Comment:
```suggestion
    Patch.addLeft(t, "/* => */ ")
```

Reply:
I suppose the same change could be applied to the 2nd code sample for [`addRight`](https://scalacenter.github.io/scalafix/docs/developers/patch.html#addright)